netwerk/base/nsSocketTransportService2.cpp

```diff
@@ -606,10 +608,27 @@ void nsSocketTransportService::ApplyPortRemapPreference(
 }
 
 PRIntervalTime nsSocketTransportService::PollTimeout(PRIntervalTime now) {
+  // #RUN-1055
+  recordreplay::RecordReplayAssert(
+    "#RUN-1055 nsSocketTransportService::PollTImeout 10 mActiveCount=%u",
+    (unsigned) mActiveCount);
   if (mActiveCount == 0) {
     return NS_SOCKET_POLL_TIMEOUT;
   }
 
+  // #RUN-1055
+  std::ostringstream oss;
+  for (uint32_t i = 0; i < mActiveCount; ++i) {
+    if (i > 0) {
+      oss << ",";
+    }
+    oss << mActiveList[i].mFD->identity;
```

Comment:
This probably needs to use NumberToStringRecordReplayWorkaroundForWindows -- we have an annoying problem where ostringstream on numbers doesn't work right on windows when replaying.  (This will probably get fixed as part of the chromium windows work coming up, I'm not looking forward to sprinkling calls to a function like this around chromium and v8.)